index.js

```diff
@@ -21,7 +21,7 @@ Boston, MA 02111-1307 USA.
 var plugins = [ "validator", "browserid-auth", "facebook", "featured", "anti-abuse",
 				"threader", "thread-color", "authorizer", "redis-storage", "storage",
 				"entityloader", "irc", "twitter", "jws", "censor", "email", "superuser", "search", "sitemap",
-				"push-notification", "google" ];
+				"push-notification", "google", "testauth" ];
```

Comment:
No testauth here.

This is the list of modules to be started on all systems. Test auth shouldn't be started on production, which is why it is started separately in lines 54 - 56.